### PR TITLE
Add new welcome page for picture doc [ci skip]

### PR DESCRIPTION
**Pull request by joshhus**
## Description

Adding Welcome page as part of the big picture doc.
## Motivation and Context

Users say big picture doc on blockchain is lacking.
## How Has This Been Tested?

Viewed on branch.
## Checklist:

- [x] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff)
- [x] Either no new documentation is required by this change, OR I added new documentation
- [x] Either no new tests are required by this change, OR I added new tests
- [] I have run [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports), [go vet](https://golang.org/cmd/vet/), and [golint](https://github.com/golang/lint). I have cleaned up all valid errors and warnings in code I have added or modified. These tools may generate false positives. Don't be worried about ignoring some errors or warnings. The goal is clean, consistent, and readable code.

Signed-off-by: Josh Horton joshh@us.ibm.com

### CODE REVIEW

File: docs/Intro/welcome.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Welcome to Hyperledger fabric
+
+The Linux Foundation Hyperledger Project(1) is the result of collaboration between project members(2) 
+who share the common goal of designing a business-ready, open-source standard for blockchain networks. 
+Extensive research, planning and implementation by this community has led to the Linux Foundation adopting 
+Hyperledger fabric as its open source standard for blockchain network architecture.
+
+The Hyperledger fabric blockchain standard is a base architecture, with built-in functionality, security
```
Comment (christo4ferris) [inline docs/Intro/welcome.md:8]: this work is not a standards development effort. While we may hope that fabric reaches de facto standard status in terms of market acceptance and adoption, it is not a standard.

File: docs/Intro/welcome.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+The Hyperledger fabric blockchain standard is a base architecture, with built-in functionality, security 
+and platform independence for blockchain networks. The open standard facilitates adoption by any public or 
+private organization or enterprise, and in turn by their constituent members.
+
+References:  
+(1) Hyperledger Project: https://www.hyperledger.org/  
+(2) Hyperledger Project members: https://www.hyperledger.org/about/members
+
+## Audience
```
Comment (christo4ferris) [inline docs/Intro/welcome.md:16]: So, basically everyone. This is far too broad. I think that ideally you would direct specific audiences to specific sections that have relevance to their interests.

File: docs/Intro/welcome.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+The audience for this Hyperledger fabric documentation includes members of any public or private group, 
+organization or enterprise who are interested in joining the Linux Foundation Hyperledger Project and 
+adopting its open standard for their business networks. The organizational roles that this audience fulfills
```
Comment (christo4ferris) [inline docs/Intro/welcome.md:20]: not a standard

**Comment by christo4ferris** [comment]
you need to enclose your email address in angle brackets eg joshh@us.ibm.com for the bot to recognize. No need to update anything, just for future reference.

As for this PR, we should discuss, because as written this should not be merged without changes, I have made specific comments, but generally, it needs to be a bit less hyperbolic.

**Comment by ghost** [comment]
Can one of the admins verify this patch?

**Comment by joshhus** [comment]
A few comments on this: 
1) Most of us don't know who the community members are well enough to target specific users for their specific interests. Such a method would also be excluding "unknown" members of the community who might want to see something / have interest in a specific doc topic.
2) Do we want to document the current state of the fabric project, or what we plan / expect it to be? The latter approach is what is typically done, for various practical reasons.
3) Regarding "far too broad," that level of doc is exactly what some users say is lacking - big picture, first-time in to the blockchain concept doc. 
4) Who rejects a doc proposal? What's the process for accepting or rejecting one, such as this one.

**Comment by christo4ferris** [comment]
@joshhus can this be closed?

**Comment by joshhus** [comment]
It depends - do we / the community agree with the conclusion in this PR that big-picture blockchain doc is not needed for the Hyperledger Fabric and / or the Project. Assuming such agreement, this can be closed. If there is not agreement, then some proposal / direction about how to publish such doc without violating "standards" is needed. @ibmmark

**Comment by christo4ferris** [comment]
@joshhus I have started refactoring and enhancing the documentation in #2150. You are welcome to pile on once that PR has been merged and recommend suggested enhancements, as are others, of course. Personally, I think we need to evolve into fabric/application/chaincode developer's guide(s), operations guide and API specifications (which should be **generated** content (from the source) wherever possible. A high-level document might be useful but the priority should be to improve the approachability of the software for users. Hence, whatever we can do to make that experience wonderful, and to improve the time to success and WOW! is OKAY by me.

**Comment by christo4ferris** [comment]
@joshhus I am going to close this. We can explore any doc changes in the context of readthedocs. Thanks

**Pull request closed without merging**